src/main/java/ch/aaap/harvestclient/domain/pagination/PaginatedList.java

```diff
@@ -55,6 +56,9 @@
 
     List<Expense> getExpenses();
 
+    @SerializedName("results")
+    List<TeamReport> getTeamReport();
```

Comment:
this will not work once we want to implement other types of reports.
We need to think of a different strategy